frontend/src/metabase-types/api/mocks/settings.ts

```diff
@@ -130,9 +131,12 @@ export const createMockSettingDefinition = (
   ...opts,
 });
 
-export const createMockSettings = (opts?: Partial<Settings>): Settings => ({
+export const createMockSettings = (
+  opts?: Partial<Settings | EnterpriseSettings>,
+): EnterpriseSettings => ({
```

Comment:
EnterpriseSettings is Settings + some optional other types, so ti shouldn't create issues
https://github.com/metabase/metabase/blob/2550abbbc1d11755a6a0cf35ecdfb396c48a0c29/enterprise/frontend/src/metabase-enterprise/settings/types.ts#L12-L19